packages/storybook8/stories/controlsUtils.ts

```diff
@@ -474,7 +474,8 @@ export const controlsToAdd = {
       if: { arg: 'allowRawObjectInput', truthy: false },
       name: 'Inject Max # of Custom Buttons'
     }
-  }
+  },
+  richTextEditor: { control: 'boolean', defaultValue: false, name: 'Enable rich text editor' }
```

Comment:
```suggestion
  richTextEditor: { control: 'boolean', name: 'Enable rich text editor' }
```

Reply:
To set default value, you will need to update the `args` list in the index.stories.tsx. But since you are defaulting to false, I don't think you need to do anything